src/unix/process.c

```diff
@@ -185,10 +185,13 @@ int uv__make_pipe(int fds[2], int flags) {
  * Used for initializing stdio streams like options.stdin_stream. Returns
  * zero on success. See also the cleanup section in uv_spawn().
  */
-static int uv__process_init_stdio(uv_stdio_container_t* container, int fds[2]) {
+static int uv__process_init_stdio(const uv_process_options_t* options, int (*pipes)[2], int i) {
```

Comment:
Please make sure lines stay <= 80 columns. The way to fix it here is to line up the second and third arguments on lines of their own.

Also, if you're only using `options->stdio`, then I'd change this to:
```suggestion
static int uv__process_init_stdio(const uv_stdio_container_t stdio[], int (*pipes)[2], int i) {
```
And index with `stdio + i` and `stdio[j]` everywhere.

Reply:
:heavy_check_mark: Done.